### PR TITLE
backend: Search Improvements

### PR DESCRIPTION
**Pull request by Neodia**
* search query param is now called search_simple_query_string
* Ref: cern-sis/cern-academic-training#36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +33,12 @@ $ yarn install
...
-$ python manage.py migrate
-$ python manage.py runserver
-$ python  manage.py opensearch index rebuild
-$ python  manage.py opensearch document index
+cd backend
+docker-compose up -d
+poetry install
+python manage.py migrate
+python  manage.py opensearch index rebuild
+python  manage.py opensearch document index
```
Comment (drjova) [inline README.md:41]: python manage.py runserver

File: backend/cds/views.py [hunk 2]
```diff
@@ -33,40 +33,41 @@ class LectureDocumentView(DocumentViewSet):
...
+        "subject_category": None
+    }
+
+    simple_query_string_search_fields = {
+        "title": {"boost": 5},
+        "abstract": {"boost": 1},
+        "type": {"boost": 5},
+        "keywords": {"boost": 6},
+        "sponsor": {"boost": 10, "proximity": 3},
+        "speaker": {"boost": 10},
```
Comment (drjova) [inline backend/cds/views.py:68]: shouldn't be also `proximity: 3`?

Comment (Neodia) [reply backend/cds/views.py:68]: Should be nothing to be honest, "proximity" was a test of mine but it doesn't do anything, I'll remove

Comment (drjova) [reply backend/cds/views.py:68]: 👍 didn't check the docs

**Comment by drjova** [comment]
@Neodia commit message "backend: search improvements" and please add a ref to the issue

File: README.md [hunk 3]
```diff
@@ -33,13 +33,13 @@ $ yarn install
...
```
Comment (drjova) [inline README.md:41]: hmm there are extra spaces between `python` and `manage.py`

File: README.md [hunk 4]
```diff
@@ -53,9 +53,9 @@ $ python manage.py loaddata users.json
 ### Add data
 
 ```bash
-$ python manage.py loaddata lectures.json
-$ python  manage.py opensearch index rebuild
-$ python  manage.py opensearch document index
+python manage.py loaddata lectures.json
+python  manage.py opensearch index rebuild
+python  manage.py opensearch document index
```
Comment (drjova) [inline README.md:58]: ditto

**Review by drjova — APPROVED** [review]
LGTM 🚀

**Pull request merged**